tslint.json

```diff
@@ -0,0 +1,34 @@
+{
+    "extends": [
+        "tslint:recommended",
+        "tslint-no-unused-expression-chai"
+    ],
+    "rules": {
+        "max-line-length": {
+            "options": [
+                240
+            ]
+        },
+        "new-parens": false,
+        "no-arg": true,
+        "no-bitwise": false,
+        "no-conditional-assignment": false,
+        "no-consecutive-blank-lines": false,
+        //"no-any": [true, {"ignore-rest-args": true}],
```

Comment:
we are aware we have disabled the no-any ... but can be incremantally enabled ...